VDL2/main.tex

```diff
@@ -1,11 +1,94 @@
-\input{../includes/preamble}
-\title{VDL2}
```

Comment:
The preamble already includes all the authors pls use it so we do not have to write them over and over again